docs/mkdocs.yml

```diff
@@ -31,6 +31,7 @@ nav:
         - app delete: 'cmd/aad/app/app-delete.md'
         - app get: 'cmd/aad/app/app-get.md'
         - app set: 'cmd/aad/app/app-set.md'
+        - app open: 'cmd/aad/app/app-open.md'
```

Comment:
Let's sort the TOC alphabetically to make it easier for our users to find commands